fdiag/templates/ice_integrals_combined_nocomparison.ipynb

```diff
@@ -161,18 +180,27 @@
     }
    ],
    "source": [
-    "plt.figure(figsize=figsize)\n",
+    "nrows, ncol = define_rowscol(input_paths, columns, )\n",
```

Comment:
define_rowscol is called with argument columns. However columns is not defined beforehand.
Maybe it can be be `len(input_paths)` by default.